examples/containers/docker-compose-exporter.yml

```diff
@@ -0,0 +1,20 @@
+---
+# This is a simple compose stack to handle Couchbase Server 6 Prometheus monitoring using the exporter.
```

Comment:
Hm, this will need documenting - you need to run this _alongside_ the existing stack docker-compose (so in a sense that comment is slightly misleading, because this isn't the whole stack, just an optional piece of it)

Reply:
Yeah I'll keep it simple I think and just say to run it manually. You can run both together by chaining `-f` arguments:
`docker-compose up -f docker-compose-yml -f docker-compose-exporter.yml`